tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -260,6 +260,48 @@ public TezConfiguration(boolean loadDefaults) {
   public static final String TEZ_TASK_LOG_LEVEL = TEZ_TASK_PREFIX + "log.level";
   public static final String TEZ_TASK_LOG_LEVEL_DEFAULT = "INFO";
 
+  /**
+   * By this option, user can easily override the logging pattern which is applied in
+   * TezContainerLogAppender in AM, regardless of the environmental settings.
+   */
+  @ConfigurationScope(Scope.AM)
+  @ConfigurationProperty
+  public static final String TEZ_LOG_PATTERN_LAYOUT_AM = TEZ_AM_PREFIX + "log.pattern.layout";
+
+  /**
+   * By this option, user can easily override the logging pattern which is applied in
+   * TezContainerLogAppender in tasks, regardless of the environmental settings.
+   */
+  @ConfigurationScope(Scope.VERTEX)
+  @ConfigurationProperty
+  public static final String TEZ_LOG_PATTERN_LAYOUT_TASK = TEZ_TASK_PREFIX + "log.pattern.layout";
```

Comment:
Is there any default value for this? (e.g If the user wants to reset it back to old value)

Reply:
good catch, user should be able to turn this off by setting it to an empty string, I'm reflecting this in the configuration